deltacat/compute/compactor_v2/deletes/utils.py

```diff
@@ -147,10 +147,11 @@ def prepare_deletes(
     """
     if not input_deltas:
         return PrepareDeleteResult(input_deltas, [], None)
+    logger.info(f"Preparing deletes for Partition-{params.source_partition_locator}")
     assert all(
         input_deltas[i].stream_position <= input_deltas[i + 1].stream_position
         for i in range(len(input_deltas) - 1)
-    ), "Uniform deltas must be in non-decreasing order by stream position"
+    ), f"Uniform deltas of length: {len(input_deltas)} must be in non-decreasing order by stream position"
```

Comment:
Is there a way to repro error faster?

Reply:
I'm attempting reproduction but that is not dependent on this logging change